pyramid_orb/services/collection.py

```diff
@@ -13,24 +13,17 @@ def __init__(self, request, collection, parent=None, name=None):
         self.collection = collection
 
     def __getitem__(self, key):
-        # look for a view
```

Comment:
view's aren't going to be done this way anymore...they should just be exposed like any other model since they are different results anyway